addons/common/backwards_comp.sqf

```diff
@@ -71,3 +71,11 @@ CBA_fnc_intToString = {
     if (isNil "_int") exitWith {""};
     str _int
 };
+
+CBA_fnc_inArea = {
```

Comment:
The problem with that is, that the function will appear in the config viewer. Deprecated functions shouldn't appear there in my opinion.